finetune/util/in_memory_finetune.py

```diff
@@ -35,8 +37,10 @@ def after_create_session(self, session, coord):
     def _evaluate(self, session):
         try:
             with tf.Graph().as_default():
-                from finetune import Classifier
-                model = Classifier(**self._config_to_finetune)
+                # from finetune import Classifier
```

Comment:
Little sketchy but I get what's going on.  Before merging to master we should fix this but fine for now.

Reply:
Which bit is sketchy?

Reply:
yep agreed, I'll add a todo